src/main/java/bookmarked/command/ReturnCommand.java

```diff
@@ -30,14 +33,16 @@ public class ReturnCommand extends Command {
      * @param listOfBooks  The list of books from which a book will be returned.
      * @param bookDataFile The data file where books are stored.
      */
-    public ReturnCommand(String[] commandParts, ArrayList<Book> listOfBooks, File bookDataFile) {
+    public ReturnCommand(String[] commandParts, ArrayList<Book> listOfBooks, File bookDataFile,
+                         ArrayList<User> listOfUsers, File userDataFile) {
         assert listOfBooks != null : "list of books should not be empty";
         assert commandParts != null : "commandParts should not be null";
         assert commandParts.length > 1 : "commandParts should contain at least the command and the book name";
         this.bookName =  String.join(" ", List.of(commandParts).subList(1, commandParts.length));
         this.listOfBooks = listOfBooks;
         this.listOfUsers = listOfUsers;
         this.bookDataFile = bookDataFile;
+        this.userDataFile = userDataFile;
```

Comment:
Nice method of creating a separate file for storing user data instead of storing in same file as book data